test/aircraft/FlightManagementSystem/Fms.spec.js

```diff
@@ -477,15 +510,7 @@ ava('.replaceDepartureProcedure() replaces the currentLeg with the new route', (
     t.true(fms.legCollection.length === 1);
 });
 
-ava('.replaceArrivalProcedure() returns early if the nextRouteString matches the current route', (t) => {
-    const routeStringMock = ARRIVAL_AIRCRAFT_INIT_PROPS_MOCK.route;
-    const fms = buildFmsMock();
-    const _findLegIndexForProcedureTypeSpy = sinon.spy(fms, '_findLegIndexForProcedureType');
-
-    fms.replaceArrivalProcedure(routeStringMock, runwayAssignmentMock);
-
-    t.false(_findLegIndexForProcedureTypeSpy.called);
-});
+ava.todo('.replaceArrivalProcedure() updates the current runway assignment');
```

Comment:
it would be still valid, a separate thing to be done, like with what we'd need to do to fix #267.